tests/integration-tests/tests/update/test_update.py

```diff
@@ -749,3 +761,295 @@ def _test_update_queue_strategy_with_running_job(
     _check_queue_ami(cluster, ec2, pcluster_ami_id, "queue1")
     _check_queue_ami(cluster, ec2, pcluster_copy_ami_id, "queue2")
     assert_compute_node_states(scheduler_commands, queue1_nodes, "idle")
+
+
+@pytest.mark.usefixtures("instance")
+def test_dynamic_file_systems_update(
```

Comment:
I think we should test the dynamic file system update in the following scenarios:
1. QueueUpdateStrategy=DRAIN
2. QueueUpdateStrategy=TERMINATE
3. QueueUpdateStrategy=COMPUTE_FLEET_STOP

If you agree with that, I think this test should be parameterized by that strategy.

Reply:
This test takes long time to run, different queue update strategies are already tested in unit tests, it is also covered in test_queue_update_strategy integration test. I don't think we should test the combinations of each parameter here

Reply:
That's great, thanks for clarifying it. Let's proceed with your solution.